dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/util/PeriodOffsetUtils.java

```diff
@@ -133,7 +133,7 @@ public static Period shiftPeriod( Period period, int periodOffset )
         }
         else
         {
-            p = periodType.getPreviousPeriod( period, periodOffset );
+            p = periodType.getPreviousPeriod( period, -periodOffset );
```

Comment:
I would prefer to use `Math.abs` to get the absolute number here to make it more explicit what is going on.

Reply:
I thought the same after a while :D

Reply:
@jimgrace do you think you could do the above fix so that we can get this one merged?

Reply:
Done. Good suggestion. That was my C programmer brain writing the code... :)